{{cookiecutter.vscode_config_path}}/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          enforce: "pre",
+          use: ["source-map-loader"],
+        },
+      ],
```

Comment:
@3coins I think this one should be part of `@jupyterlab/builder`. Would you mind opening a PR for that?

Reply:
Sure, thanks for looking into this.

Reply:
Opened PR here.
https://github.com/jupyterlab/jupyterlab/pull/13765